src/be_txn.erl

```diff
@@ -57,10 +61,11 @@ to_json(blockchain_txn_security_coinbase_v1, T, _Ledger) ->
 to_json(blockchain_txn_oui_v1, T, _Ledger) ->
     #{<<"owner">> => ?BIN_TO_B58(blockchain_txn_oui_v1:owner(T)),
       <<"addresses">> => [binary_to_list(Addr) || Addr <- blockchain_txn_oui_v1:addresses(T)],
```

Comment:
What kind of addresses are these? Are they router_addresses? If so, BIN_TO_B58?

Reply:
Added in 8bec535